frontend/src/App.svelte

```diff
@@ -83,9 +83,8 @@
       const result = await response.json();
       result.id = token;
 
-      ownedTokens.push(result);
+      ownedTokens = [...ownedTokens, result];
```

Comment:
This is a regression. Push and reassign _is_ the svelte way, but recent React converts feel better about spread assignment. It's significantly slower